ee/debug/checkups/install.go

```diff
@@ -55,52 +54,16 @@ func gatherInstallationLogs(z *zip.Writer) error {
 		return nil
 	}
 
-	out, err := z.Create("macos-var-log-install.log")
-	if err != nil {
-		return fmt.Errorf("creating macos-var-log-install.log in zip: %w", err)
-	}
-
-	installLog, err := os.Open("/var/log/install.log")
-	if err != nil {
-		return fmt.Errorf("opening /var/log/install.log: %w", err)
-	}
-	defer installLog.Close()
-
-	if _, err := io.Copy(out, installLog); err != nil {
-		return fmt.Errorf("writing /var/log/install.log contents to zip: %w", err)
-	}
-
-	return nil
+	return addFileToZip(z, "/var/log/install.log")
```

Comment:
I think this has the inadvertent effect that it's going to cause the end files to be named something else, is that correct? I don't think that's inherently wrong, but I want to note it.

I think we might even want it, since I'd thought about the idea of moving all the files into a dedicated file directory, instead of mini zip files for each flare.

Reply:
That's correct the file will always be install.log instead of varying for macos